packages/imperative/__tests__/Imperative.test.ts

```diff
@@ -356,6 +357,10 @@ describe("Imperative", () => {
                     envConfig = mocks.EnvironmentalVariableSettings.read(Imperative.envVariablePrefix);
                 });
 
+                it("should know what the envVariablePrefix is [DEPRECATED]", async () => {
```

Comment:
Besides from specifying the `envVariablePrefix` as part of the application's configuration object (in `imperative.ts`), I don't think we allowed another way to set this property since there was no `setter` defined for it.
Any existing non-standard workarounds they may be doing today to redefine it, e.g. `(Imperative as any).envVariablePrefix = xyz`, should still work even after the deprecation (and even after the removal).